src/tests/ftest/pool/attribute.py

```diff
@@ -118,21 +121,23 @@ def test_pool_large_attributes(self):
         """
         self.add_pool()
         attr_dict = self.create_data_set()
-
         try:
             self.pool.pool.set_attr(data=attr_dict)
             size, buf = self.pool.pool.list_attr()
 
             self.verify_list_attr(attr_dict, size.value, buf)
 
             results = {}
-            results = self.pool.pool.get_attr(list(attr_dict.keys()))
+            results = self.pool.pool.get_attr(list(attr_dict.values()))
```

Comment:
The pool `get_attr` function should be provided a list of attribute names, not values:
```suggestion
            results = self.pool.pool.get_attr(list(attr_dict.keys()))
```